lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -768,14 +777,14 @@ struct ConvertLayoutOpConversion
               numElemsPerSwizzlingRow, true);
 
           Value addr = gep(elemPtrTy, smemBase, offset);
-          Value data0 = rewriter.create<triton::nvgpu::CvtPackOp>(
-              loc, i32_ty, inVals[elemIdx + 1], inVals[elemIdx + 0]);
-          Value data1 = rewriter.create<triton::nvgpu::CvtPackOp>(
-              loc, i32_ty, inVals[elemIdx + 3], inVals[elemIdx + 2]);
-          Value data2 = rewriter.create<triton::nvgpu::CvtPackOp>(
-              loc, i32_ty, inVals[elemIdx + 5], inVals[elemIdx + 4]);
-          Value data3 = rewriter.create<triton::nvgpu::CvtPackOp>(
-              loc, i32_ty, inVals[elemIdx + 7], inVals[elemIdx + 6]);
```

Comment:
Should we remove `triton::nvgpu::CvtPackOp` from the dialect then ? It's only used in the code you changed

Reply:
ah yes sorry I forgot

Reply:
done